src/org/zaproxy/zap/extension/communityScripts/resources/Messages_nl_NL.properties

```diff
@@ -2,4 +2,4 @@
 # This file defines the default (English) variants of all of the internationalised messages
 
 communityScripts.name=Community Scripts
-communityScripts.desc=Community Scripts from https\://github.com/zaproxy/community-scripts
+communityScripts.desc=Commmunity Scripts van https\://github.com/zaproxy/community-scripts
```

Comment:
3 m again....doesn't make sense. Checked crowdin suggestions and Google translate